### DISCUSSION TITLE
Capitated Agreements With Underlying Fee Schedules

### DISCUSSION THREAD

**Discussion by angelamarie0975** [CMSgov/price-transparency-guide#21] [General]
Asking the community to see how others are handling this situation. I have added this question under the Issues section as well. How exactly should the JSON file be laid out when the provider is capitated and we need to also display the underlying fee schedule per procedure. In the capitation example, the layout flow is plan/network arrangement (Capitation)/negotiated price/covered services. How do we apply both the capitated agreement and the underlying fee schedule under the In-Network Object for a capitated negotiated agreement? Would we we have 2 In-Network Objects for the same providers - one listed as Capitation negotiation agreement and the same provider under a FFS negotiation agreement for the underlying fee schedule for the same procedures listed in the Covered Services Object in the Capitation agreement? 

Blurb from white papers = The “Underlying Fee-Schedule Rate,” which is the rate a Plan or Issuer uses to determine a Member’s cost-sharing liability for an Item or Service, if the rate is different from the Negotiated Rate or Derived Amount.  45 C.F.R. § 147.210(a)(2)(xxii).  Thus, for example, a Plan or Issuer that pays an InNetwork Provider a capitated rate might use a fee schedule to determine the amount on which Member coinsurance should be based.  A fee schedule for a Member’s 30% coinsurance might value an office visit at $100, in which case the Member pays $30 coinsurance.  **_In this example, the Plan or Issuer would report the capitated rate as the Negotiated Rate and the Underlying Fee Schedule Rate as $100._**

**Comment by tdfow** [comment]
#10 - related topic

**Comment by shaselton-usds** [comment]
For a capitated arrangements where the negotiated rate for an item or service that does not reflect the patients cost sharing liability,  you would include two `negotiated_price` objects with the negotiated arrangement with the provider (this might be a PMPM value) and the underlying fee schedule in which the cost sharing liability is based on.

```
{
    "negotiation_arrangement": "capitation",
    "name": "...",
    "billing_code_type": "...",
    "billing_code_type_version": "...",
    "billing_code": "...",
    "description": "...",
    "negotiated_rates": [{
      "provider_groups": [{
        "npi": [...],
        "tin":{
          "type": "...",
          "value": "..."
        }
      }],
      "negotiated_prices": [{
        "negotiated_type": "negotiated",
        "negotiated_rate": 123.45,
        "expiration_date": "2022-01-01",
        "service_code": ["18", "19", "11"],
        "billing_class": "professional"
      },{
        "negotiated_type": "fee schedule",
        "negotiated_rate": 1230.45,
        "expiration_date": "2022-01-01",
        "service_code": ["18", "19", "11"],
        "billing_class": "professional"
      }]
  }
```
